dascore/transform/differentiate.py

```diff
@@ -72,17 +111,27 @@ def differentiate(
     --------
     >>> import dascore as dc
     >>> patch = dc.get_example_patch()
+    >>>
+    >>> # Example 1
     >>> # 2nd order stencil for 1st derivative along time dimension
```

Comment:
All examples here use `order=2`, so I suggest using the same docs for `order` to avoid any confusion (here we have "2nd order stencil" while we do not have it in other examples.)

Reply:
Sorry, I am not sure what you mean here, could you elaborate?